src/main/scala/org/broadinstitute/dsde/workbench/sam/openam/PostgresAccessPolicyDAO.scala

```diff
@@ -901,3 +926,9 @@ final case class AncestorResourceTable(override val tableName: String) extends S
   // need to specify column names explicitly because this table does not actually exist in the database
   override val columnNames: Seq[String] = Seq("resource_parent_id")
 }
+
+final case class ChildResourceRecord(childId: ResourcePK)
+final case class ChildResourceTable(override val tableName: String) extends SQLSyntaxSupport[ChildResourceRecord] {
```

Comment:
Are these used anywhere?

Reply:
Nope! Actually just pushed up a commit to remove them. They were leftover from when I thought we wanted all descendants of a resource, not just direct children